stories/core/Table.stories.tsx

```diff
@@ -847,11 +850,13 @@ export const LazyLoading: Story<Partial<TableProps>> = (args) => {
 
   return (
     <Table
+      useVirtualization
       columns={columns}
       emptyTableContent='No data.'
       onBottomReached={onBottomReached}
       isLoading={isLoading}
       {...args}
+      style={{ height: 440 }}
```

Comment:
Maybe change to `90vh` or use decorator as in other stories.

Reply:
changed